charts/developers-italia-api/Chart.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: v2
+name: developers-italia-api
+description: The API of Developers Italia
+
+# A chart can be either an 'application' or a 'library' chart.
+#
+# Application charts are a collection of templates that can be packaged into versioned archives
+# to be deployed.
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.1.0
+
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application. Versions are not expected to
+# follow Semantic Versioning. They should reflect the version the application is using.
+# It is recommended to use it with quotes.
+appVersion: "v0.1.1-pre"
+
```

Comment:
Can we leave `appVersion` out, so we don't have to bump it every time?

We usually do and Keel should already take care of upgrading.

Reply:
We can configure Keel directly in the `values.yml` override that will be in https://github.com/teamdigitale/devita-infra-kubernetes (see https://keel.sh/docs/#helm-example), and release this chart to the GitHub registry with appVersion tied to the latest stable Docker image.

Reply:
@tensor5 sounds great :+1: